### PR TITLE
BNF

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Corrections, new record

### CODE REVIEW

File: ParisBNF/abb/BNFabb137.xml [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno>BnF Éthiopien d'Abbadie 137</idno>
+                        <idno facs="https://archivesetmanuscrits.bnf.fr/ark:/12148/cc94965m">BnF Éthiopien d'Abbadie 137</idno>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb137.xml]: the second idno should be in an `<altIdentifier>`

File: ParisBNF/abb/BNFabb137.xml [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+                                    <dimensions type="outer" unit="mm">
+                                      <height>210</height>
+                                      <width>180</width>
+                                    </dimensions>
+                                </extent>
+                                <foliation>
+                                </foliation>
+
+                                <collation>
+                                </collation>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb137.xml]: empty elements could be removed

File: ParisBNF/abb/BNFabb137.xml [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+                     </person>
+                </listPerson>
+                </particDesc>
+            <textClass>
+                <keywords>
+                    <term key="Bible"/>
+                    <term key="OldTestament"/>
+                    <term key="Religion"/>
+                    <term key="Gon"/>
+                    </keywords>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb137.xml]: if it is a manuscript belonging to the Christian tradition, ChristianLiterature should be added

File: ParisBNF/abb/BNFabb137.xml [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+            </textClass>
+            <langUsage>
+                  <language ident="de">French</language>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb137.xml]: French or German? :)

File: ParisBNF/abb/BNFabb203.xml [hunk 5]
```diff
@@ -40,49 +40,119 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                </colophon>
                            </msItem>
                          </msContents>
 
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
-                           <material key="parchment">Parchment.</material>
+                           <material key="parchment">Oriental paper.</material>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb203.xml]: then the value "parchment" should be corrected

File: ParisBNF/abb/BNFabb203.xml [hunk 6]
```diff
@@ -92,18 +162,126 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                            <handNote xml:id="h1" script="Ethiopic">
                               <seg type="script">19th century.</seg>
                               <seg type="ink">Black, red.</seg>
-                              <seg type="rubrication">Initial lines in many chapters,
-                                meant to be rubricated, are omitted.</seg>
-                              <date>19th century.</date>
+                              <seg type="rubrication">Incipits of the texts,
+                              the word for "chapter" and chapter numbers.</seg>
+                              <desc>Not very regular.</desc>
+                              <date>Dated to 1842 AD.</date>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb203.xml]: the date could be marked up

File: ParisBNF/abb/BNFabb203.xml [hunk 7]
```diff
@@ -92,18 +162,126 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                  <item xml:id="e2">
+                                     <desc>Stamp of the library,
+                                       on the <ref target="#b5">left pastedown</ref>.</desc>
+                                      </item>
+
+
+                                  <item xml:id="e3">
+                                     <locus target="#1r"/>
+                                      <desc type="OwnershipNote" xml:lang="gez"/>
+                                    <q xml:lang="gez">ዝመጽሐፍ᎓ ዘአባ᎓ ዮስጦስ᎓</q>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb203.xml]: the name could also be marked up in fidal

File: ParisBNF/abb/BNFabb137.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb137.xml]: Done

Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb137.xml]: Done.

File: ParisBNF/abb/BNFabb137.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb137.xml]: Filled.

File: ParisBNF/abb/BNFabb137.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb137.xml]: Done.

File: ParisBNF/abb/BNFabb137.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb137.xml]: Corrected.

File: ParisBNF/abb/BNFabb203.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb203.xml]: Done.

File: ParisBNF/abb/BNFabb137.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb137.xml]: Done.

**Pull request merged**